### ISSUE TITLE
Optional block SSL handshake error bypass

### ISSUE THREAD

**Issue by rsilva-freiheit** [ThexXTURBOXx/flutter_web_auth_2#148]
Hello @ThexXTURBOXx 

First of all, thank you for your excellent work!

## Is your feature request related to a problem? Please describe.
Currently, users can bypass the SSL handshake errors in iOS, potentially compromising their logins by clicking on the “Details” button and opening the webview anyway.

## Describe the solution you’d like
I propose implementing an argument or parameter in the controller to prevent users from bypassing the handshake exception.

## Describe alternatives you’ve considered
Currently, I have a check that prevents the webview from opening unless the SSL handshake is successful but this is still not ideal

**Comment by ThexXTURBOXx** [comment]
I am not too sure whether it is within my abilities to implement this feature on Apple devices since I do not own any.
Also, it seems kind of reasonable to me that one can still bypass these exceptions by default as certificates often are forgotten to be renewed.
Hence, I would say that special handling this is within the hands of the developer himself and should not be handled by this package.
A quick search showed me this dedicated (but probably rather outdated) package: https://pub.dev/packages/ssl_checker
It probably makes more sense to use solutions like these to properly address this issue (even cross-platform!).

**Comment by rsilva-freiheit** [comment]
I agree that it’s reasonable to allow bypassing these exceptions by default, which is why I suggested making it an optional parameter. However, having the WebView UI block the bypass would simplify integration, promote a more secure implementation, and encourage better security practices by simple just allowing it to be set.

That said, I understand that implementing this functionality cross-platform adds complexity.

Thank you for clarifying!
